src/lang/checkers/Cashflow.ml

```diff
@@ -954,9 +942,8 @@ module ScillaCashflowChecker
                 (* Error *)
                 [[ ( Inconsistent , List.map ~f:(fun _ -> Inconsistent) args_tags ) ]] in
           (c_r_sigs, c_as_sigs)
-      | "substr"
-      | "schnorr_sign"
-      | "schnorr_verify" ->
+      | Builtin_substr
+      | Builtin_schnorr_verify ->
```

Comment:
I just realised that this pattern-match must be missing some cases.

The point of replacing the string representations with an internal ADT was to have the type-checker ensure that we are handling all possible built-ins everywhere. However, this only works if we don't use a wildcard pattern, as we do at the end of this match.

By the looks of it there is a handful of built-ins that we are not calculating signatures for, so I think it's better to merge this PR, and create an issue on the missing signatures.